loafwallet/NonUSTabBarViewController.swift

```diff
@@ -0,0 +1,430 @@
+//
+//  NonUSTabBarViewController.swift
+//  loafwallet
+//
+//  Created by Kerry Washington on 5/9/21.
+//  Copyright © 2021 Litecoin Foundation. All rights reserved.
+//
+ 
+import UIKit
+import Foundation 
+
```

Comment:
Actually, I would prefer to refactor to a `TabView` and set a conditional layout .  The problem is this code is quite old and we have limited time to put in the changes.
The app leverages a weird view stack, which is not typical but was inherited from the Bread fork.

Reply:
Let me review the switch statements...👀

Reply:
@kcw-grunt yeah agreed with @JasperHeist to have a BaseTabBarViewController since the both `TabBarViewController` and `NonUSTabBarViewController` are almost the same. But as you said it could take time . So for now let's add it into the technical debt issues